src/components/navigation/NavBar.js

```diff
@@ -21,7 +21,7 @@ export default class NavBar extends React.PureComponent {
   };
 
   renderCrumbs() {
-    const crumbs = this.props.routeTree.reverse().slice(1);
+    const crumbs = [...this.props.routeTree].reverse().slice(1);
```

Comment:
This hack-y spread operator trick is to prevent the prop from being mutated. It will be fixed by addressing #238.